src/common/platform/posix/cocoa/i_video.mm

```diff
@@ -102,6 +102,7 @@ - (void)enterFullscreenOnZoom
 EXTERN_CVAR(Bool, vk_debug)
 
 CVAR(Bool, mvk_debug, false, 0)
+CVAR(Bool, vid_macfullscreen, true, CVAR_ARCHIVE | CVAR_GLOBALCONFIG)
```

Comment:
Please name it `vid_nativefullscreen` with the default value set to false.

Reply:
Ah sorry, I thought you had said to have it enabled by default in your first comment.